src/generate/renders/tests/CliTestStep.ts

```diff
@@ -218,6 +217,9 @@ export class CliTestStep extends TemplateBase {
     }
 
     public static parameterLine(parameterNames, withChecksDef = false) {
+        if (!AzConfiguration.getValue(CodeGenConstants.az)[CodeGenConstants.useTestStepParam]) {
```

Comment:
Can we use === true to decide whether user has set the correct value ? just in case they put a lot of strange values